app/src/js/components/Requests/request.js

```diff
@@ -65,6 +69,7 @@ class RequestOverview extends React.Component {
     // This causes a repeating query for workflows cluttering up the logs.
     // Commenting out until we add applyWorkflow capability
     // this.cancelInterval = interval(this.queryWorkflows, updateInterval, true);
+    dispatch(listUsers());
```

Comment:
DAAC Staff don't have the necessary privileges for using this endpoint:
```
{"error":"No privilege"}
```

Reply:
ohh ok...what would you suggest on this? should i create a new endpoint or add a privileges for DAAC Staff to this endpoint.